var/spack/repos/builtin/packages/py-itk/package.py

```diff
@@ -3,159 +3,16 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import sys
-
-import archspec
-
 from spack.package import *
+from spack.pkg.builtin.itk import Itk
 
 
-class PyItk(PythonPackage):
+class PyItk(BundlePackage):
```

Comment:
Not sure how I feel about this. This strategy could potentially be used for dozens of packages that include both Python and non-Python components (e.g., opencv+python and py-opencv-python). Curious what others think (@tldahlgren @tgamblin)